ext/opentelemetry-ext-zipkin/tests/test_zipkin_exporter.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2018, OpenCensus Authors
```

Comment:
Unless you know better, I don't think we need to keep the OC attribution. We haven't kept it on other modules, and this wasn't copied directly.

Reply:
I'll remove it. I noticed it was still there on the jaeger module